code/blocks/AccordionBlock.php

```diff
@@ -55,4 +55,20 @@ public function getCMSFields()
 
         return $fields;
     }
+}
+
+class AccordionBlock_Controller extends Block_Controller
+{
+    public function init()
+    {
+        Requirements::javascript(THIRDPARTY_DIR . '/jquery-ui/jquery-ui.js');
+        Requirements::customScript('
```

Comment:
Would it make sense to allow for customizing the css class for the accordion? Possibly a default config but a developer could update the default if they need to in the event of a class collision in their css.